components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -680,11 +679,11 @@ impl Interpolate for BoxShadow {
     }
 }
 
-impl Interpolate for LengthOrNone {
+impl<T: Copy> Interpolate for Either<Au, T> {
```

Comment:
It doesn't have to be `Copy` (removed it and added a more-or-less generic impl).